Src/Newtonsoft.Json.UnityConverters/EnumerableVectorConverter.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json.Linq;
+using UnityEngine;
+#if PORTABLE
+using System.Reflection;
+#endif
+
+namespace Newtonsoft.Json.UnityConverters
+{
+    internal class VectorTypes
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4846771)